Tools/CMake/AMReX_Options.cmake

```
@@ -144,7 +144,8 @@ endif ()
 #
 # AMReX components selection  ================================================
 #
-option( ENABLE_EB "Build EB Code" OFF )
+cmake_dependent_option( ENABLE_EB "Build EB code" OFF
```

We could improve the doc string while changing this:
```suggestion
cmake_dependent_option( ENABLE_EB "Build with Embedded Boundary support" OFF
```